fdbclient/IdempotencyId.actor.cpp

```diff
@@ -202,4 +202,130 @@ void decodeIdempotencyKey(KeyRef key, Version& commitVersion, uint8_t& highOrder
 	reader >> commitVersion;
 	commitVersion = bigEndian64(commitVersion);
 	reader >> highOrderBatchIndex;
+}
+
+// Find the youngest or oldest idempotency id key in `range` (depending on `reverse`)
```

Comment:
[Optional] It might be useful to clarify what `reverse` means. So for example:
Find the oldest or youngest (if `reverse` is true) idempotency id key in `range`.